python-backend/app/api/mines/mine/models/mine.py

```diff
@@ -18,7 +18,7 @@ class Mine(AuditMixin, Base):
     mine_name = db.Column(db.String(60), nullable=False)
     mine_note = db.Column(db.String(300), default='')
     major_mine_ind = db.Column(db.Boolean, nullable=False, default=False)
-    deleted_ind = db.Column(db.Boolean, nullable=False, default=True)
+    deleted_ind = db.Column(db.Boolean, nullable=False, default=False)
```

Comment:
I think this could use the fetch function to pull the default from the db, which is the pattern to use going forward, instead of hardcoding the defaults into the application level

Reply:
changed.